person.rb

```diff
@@ -7,19 +8,20 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
```

Comment:
- [ ] due to setting up the default parent permission like this `parent_permission: true` the user will receive the below error when creating a person please replace it by `parent_permission = true`

![obraz](https://user-images.githubusercontent.com/39240820/184182423-ef4e57d5-59d0-46ce-847a-68d62542b483.png)

Reply:
@mikethreels If I use parent_permission = true, i get errors in rubocop linters saying its boolean that I use  :true instead
![TSE](https://user-images.githubusercontent.com/98754783/184293948-79037820-716a-4319-a509-c8b987362ea5.png)
.

Reply:
Hi @Stanleeeeee,
I wasn't aware that rubocop gave this error, in that case you will need to change it back, however make sure the bug is also fixed.